tests/isisd/test_isis_lspdb.c

```diff
@@ -82,6 +82,7 @@ static void test_lsp_build_list_nonzero_ht(void)
 
 int main(int argc, char **argv)
 {
+	struct isis *isis = NULL;
```

Comment:
This belongs with the previous commit

Reply:
Sorry, I didn't understand this comment. 

When we removed the global isis "struct isis *isis = NULL". 
While doing " make " in test/ directory it was giving an error in this file ( tests/isisd/test_isis_lspdb.c ) 
So, created this local variable inside the main of "test_isis_lspdb.c". 

Do you want this particular change in a separate PR  ?